packages/terra-framework-docs/src/terra-dev-site/doc/slider/About.1.doc.mdx

```diff
@@ -2,31 +2,91 @@ import { Notice } from "@cerner/terra-docs";
 import { Badge } from 'terra-slider/package.json?dev-site-package';
 
 import SliderPropsTable from 'terra-slider/src/Slider?dev-site-props-table';
-
 import DefaultSlider from './Examples.2/DefaultSlider?dev-site-example';
-import CustomSlider from './Examples.2/CustomSlider?dev-site-example';
-import DisabledSlider from './Examples.2/DisabledSlider?dev-site-example';
-import SliderWithOnChangeCallback from './Examples.2/SliderWithOnChangeCallback?dev-site-example';
-import SliderWithValueUpdate from './Examples.2/SliderWithValueUpdate?dev-site-example';
 
 <Badge />
 
 # Terra Slider
 
+## Table of Contents
+* [Overview](/components/cerner-terra-framework-docs/slider/about#overview)
+* [Examples](/components/cerner-terra-framework-docs/slider/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/slider/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/slider/about#implementation)
+* [Testing](/components/cerner-terra-framework-docs/slider/about#testing)
+* [Properties](/components/cerner-terra-framework-docs/slider/about#properties)
+* [Terra Standards](/components/cerner-terra-framework-docs/slider/about#terra-standards)
+
+## Overview
+
 Terra Slider provides a way to select a value from a given range by sliding a thumb along a bar to track the value.
 
 <Notice variant="caution" ariaLevel="2">
 Terra Slider is an input component used to quickly select an imprecise value within a range such as a zoom level. It does not yet include features required for entering precise values.
 </Notice>
 
+<DefaultSlider />
+
+### Features
+Supports:
+* Management of custom slider label, minimum label and maximum label.
```

Comment:
nit: Add an 'a' in front of custom
"Management of a custom slider lable, minimum label and maximum label."